### ISSUE TITLE
Convert image from png to webp not working correctly

### ISSUE THREAD

**Issue by MS893** [android/size-analyzer#7]
One of the suggestions is to convert PNG images to WEBP images which give a significant size gain if you have lot of icons and images in your app.

The problem is that this plugin gives you the possibility to convert images one by one or all at the same time, but it doesn't work because it says that the images are already opened and they are not ! So, the conversion works (webp images are created) but the deletion of the png images doesn't work, and you have to navigate into your different directories to delete manually all the png images.

The "convert all" button doesn't create nothing (no webp images created) and of course doesn't erase the png images... and there is no message to say "conversion failed" but the event log has a new error message and allows to report the bug (what I have done).

Thanks.

Version: v0.3.1

**Expected behavior**
One by one : create webp images (OK) AND delete old png images (not OK)
Convert all button : create and delete (not OK : no web,p no deletion)

**Comment by tdeck** [comment]
Thank you for reporting this issue. Are you encountering this behavior on Windows? Sounds to me like Windows file locking behavior.

**Comment by tdeck** [comment]
Hopefully there's a better way of handling it but that's one option; I think we haven't tested as extensively on Windows so we weren't aware of this issue until you reported it (thanks again).

**Comment by MS893** [comment]
I understand, I prefer Linux but Asus didn't provide drivers for Linux, so I have to use Windows.

If you can do something, it would be great... Thanks

**Comment by tdeck** [comment]
This has been fixed in release 0.4.0. I've tested it on Windows and it works.

I'm sorry it's taken so long to address this issue. The Size Analyzer was something of an experimental project, and we haven't had a team dedicated to continuing its development. Since I'm leaving Google in a couple of days, I thought I'd try to tie up these loose ends. I've also updated the README to reflect the status of the project.
